drive/utils.bal

```diff
@@ -662,3 +665,21 @@ isolated function uploadFileUsingByteArray(http:Client httpClient, byte[] byteAr
     }
 }
 
+# Utility function to retrieve File content as a binary content with its MIME type.
+# 
+# + httpClient - Drive client
+# + path - GET URI path
+# + return - `FileContent` if successful, `error` if not.
+isolated function generateRecordFileContent(http:Client httpClient, string path) returns @tainted FileContent|error {
+    http:Response response = <http:Response> check httpClient->get(<@untainted>path);
+    if (response.statusCode is http:STATUS_OK) {
```

Comment:
Usually, we get status code as int and we use "is" to check the types like string, null, int etc. So Isn't it should be  "==" ?

Reply:
I checked that both work fine. So what can be the best? any other ideas:)?

Reply:
yeah. That is good. I changed it
/Thanks